src/web_app_skeleton/tasks/db_seed_required_data.cr

```diff
@@ -4,9 +4,9 @@ require "../spec/support/boxes/**"
 # For example, you might need at least one admin user or you might need at least
 # one category for your blog posts for the app to work.
 #
-# Use `Db::CreateSampleSeeds` if your only want to add sample data helpful for
+# Use `Db::Seed::SampleData` if your only want to add sample data helpful for
 # development.
-class Db::CreateRequiredSeeds < LuckyCli::Task
+class Db::Seed::RequiredData < LuckyCli::Task
```

Comment:
I renamed this file but I'm not sure about the new name. I considered `db/seeds/required_data.cr` but that felt like a big enough change that it warranted discussion.

What is a good naming convention for these files?

Reply:
I'm fine with this class name, but could you update the file structure to match? `tasks/db/seed/required_data.cr`

Reply:
All set